packages/documentation/docs/controls/_breadcrumb_styleguide.md

```diff
@@ -0,0 +1,113 @@
+
+## Variants
+
+![Breadcrumb variants](/img/pattern_illustrations/Breadcrumb_variants.png)
+
+### Emphasis
+
+In order to vary the visual emphasis of a breadcrumbs control, a ghost variant (default) and a solid variant are available. Both variants differ in font color and fill color but not in their interaction patterns.
+
+### Item interactivity
+
+Breadcrumb items are interactive and allow the user to navigate to their respective location by pressing the item. An exception is the last item in a breadcrumb control representing the current page. By default, this item does not offer any user interaction. An interactive item variant is available which allows the user to browse to children elements of the current page. Pressing the item triggers a dropdown listing all child elements.
+
+### Content
+
+Breadcrumb items can, but don't have to, include an icon. The icon is positioned before the breadcrumb label. Icons can be included for each item or only for specific items (e.g. the root item).
+
+## Behavior in context
+
+### Population
+
+The breadcrumb should be populated location-based to reflect the hierarchy of the application and the loction of the user in it. The current location should always be included in the breadcrumb.
+
+### Overflow
+
+The breadcrumb control is set to display a limited number of items (default: 9 items). If the number of items exceeds the defined limit, excessive items are hidden within a dropdown menu at the beginning of the path. The dropdown menu is triggered by pressing the respective item. The truncation is visualized with a ellipsis.
+
+The overflow behaviour can also be triggered if the available space does not allow the complete display of the breadcrumb in one line.
+
+Truncation is applied to individual breadcrumb items if the maximum width of the breadcrumb item is exceeded. The label name is truncated with an ellipsis.
+
+<!-- add example overflow & truncation-->
+<!-- Open question: How is the limit defined for the labels? Bug fix? -->
+
+### Placement
+
+Breadcrumbs are placed at the top left side of the page/ content area. They are positioned below the header and above the page title.
+
+## States
+
+### Interactive item
+
+| State   | Description                                                                          |
+| ------- | ------------------------------------------------------------------------------------ |
+| Hover   | The hover state is indicated by font and fill color change.                          |
+| Active  | The active state is indicated by font and fill color change. Equal to pressed state. |
+| Focused | The focused state is indicated by a focus frame.                                     |
+
+<!-- focused state on interactive item available? -->
```

Comment:
Yes, focus state should be available on interactive element, is currently missing --> issue #417